### PR TITLE
Update bitmaps.md

### PR DESCRIPTION
**Pull request by sav-norem**
update bitmaps data type page to include hugo short code - condensed example and added relevant context

### CODE REVIEW

**Comment by sav-norem** [comment]
@dwdougherty the last data types docs page that needs hugo short codes at the moment

File: docs/data-types/bitmaps.md [hunk 1]
```diff
@@ -21,37 +21,30 @@ Some examples of bitmap use cases include:
...
-Suppose you have 1000 sensors deployed in the field, labeled 0-999.
-You want to quickly determine whether a given sensor has pinged the server within the hour. 
+Suppose you have 1000 cyclists racing through the country-side, labeled 0-999.
+You want to quickly determine whether a given rider has pinged the server within the hour. 
 
 You can represent this scenario using a bitmap whose key references the current hour.
 
-* Sensor 123 pings the server on January 1, 2024 within the 00:00 hour.
-```
+* Rider 123 pings the server on January 1, 2024 within the 00:00 hour. We can then confirm that rider 123 pinged the server. We can then check to see if rider 456 has pinged the server for that same hour.
```
Comment (dwdougherty) [inline docs/data-types/bitmaps.md:35]:
```suggestion
* Rider 123 pings the server on January 1, 2024 within the 00:00 hour. You can then confirm that rider 123 pinged the server. You can also check to see if rider 456 has pinged the server for that same hour.
```

**Review by dwdougherty — CHANGES_REQUESTED** [review]
Just a coupla comments.

File: docs/data-types/bitmaps.md [hunk 2]
```diff
@@ -21,37 +21,30 @@ Some examples of bitmap use cases include:
...
 > GETBIT pings:2024-01-01-00:00 123
 1
-```
-
-* What about sensor 456?
-```
 > GETBIT pings:2024-01-01-00:00 456
 0
-```
+{{< /clients-examples >}}
```
Comment (dwdougherty) [inline docs/data-types/bitmaps.md:44]:
```suggestion
{{< /clients-example >}}
```

File: docs/data-types/bitmaps.md [hunk 3]
```diff
@@ -89,15 +73,12 @@ There are three commands operating on group of bits:
...
-    > setbit key 100 1
-    (integer) 0
-    > bitcount key
-    (integer) 2
+string, instead of running for the whole length of the string. We can trivially see the number of bits that have been set in a bitmap.
+
+{{< clients-example bitmap_tutorial bitcount >}}
+> BITCOUNT pings:2024-01-01-00:00
+(integer) 1
+{{< /clients-examples >}}
```
Comment (dwdougherty) [inline docs/data-types/bitmaps.md:81]:
```suggestion
{{< /clients-example >}}
```

**Review by dwdougherty — CHANGES_REQUESTED** [review]
The closing tags weren't correct, causing build failure.

**Review by dwdougherty — APPROVED** [review]
Builds correctly. Approved.

File: docs/data-types/bitmaps.md [hunk 4]
```diff
@@ -21,37 +21,30 @@ Some examples of bitmap use cases include:
...
 See the [complete list of bitmap commands](https://redis.io/commands/?group=bitmap).
 
 
-## Examples
+## Example
 
-Suppose you have 1000 sensors deployed in the field, labeled 0-999.
-You want to quickly determine whether a given sensor has pinged the server within the hour. 
+Suppose you have 1000 cyclists racing through the country-side, labeled 0-999.
+You want to quickly determine whether a given rider has pinged a tracking server within the hour.
```
Comment (zuiderkwast) [inline docs/data-types/bitmaps.md:31]: Sorry, but I think connected sensors are more common than connected cyclists. Or maybe it's a censor on each bicycle? :)

**Comment by sav-norem** [comment]
I definitely meant it as a sensor on each bicycle! Can clarify that wording.

**Pull request merged**